config/config.rb

```diff
@@ -124,8 +124,8 @@
   configatron.beckman_precision = 2
   configatron.hamilton_precision = 2
 
-  configatron.data_sharing_contact.name = 'John Doe'
-  configatron.data_sharing_contact.email = 'foo'
+  configatron.data_sharing_contact.name = 'Datasharing'
```

Comment:
It looks like maybe this name isn't used any more.  Otherwise all looks good.

Reply:
I meant that I don't think the name is used in the code.  I haven't checked all of the code, but you removed a reference to the name in that `data_release_delay_reason:help:` text and that might have been the only reference.

Reply:
I take it that this is still referenced in the code somewhere?  I don't think it is, but I haven't gone through the code to find out.

Reply:
It is still in the configuration. configatron.data_sharing_contact.name locale string is not referenced in the code. 

They did not included the name in the help text in their request for this story; they asked only for the email to be included. I would like to keep the name in the configuration:

Previously, the contact information was for an actual person. The name of the person and the email were in the help text. It was changed to a new service contact information name: Datasharing, email: datasharing@sanger.ac.uk . Only the email is included in the help text. Although we do not use configatron.data_sharing_contact.name in the code, I think, it is still part of the configuration. 

The word "Datasharing" has a mention in a few places, but they are hardcoded; not loaded from the locale string and I think fixing them is not part of this story. Searching the word in the project will reveal several instances mainly "Contact Datasharing".

Another point to mention that the name and email are configured using the deployment project. If we have to remove the name, we need to remove it from both Sequencescape and Deployment. If we need them again, or we want to improve the code, we will need to change two projects.

I will now attempt using configatron.data_sharing_contact.name whenever Datasharing appears in other places, possibly by creating new locale strings and doing interpolation not to drop the item.